launcher/modplatform/modrinth/ModrinthAPI.h

```diff
@@ -28,30 +28,25 @@ class ModrinthAPI : public NetworkModAPI {
    public:
     inline auto getAuthorURL(const QString& name) const -> QString { return "https://modrinth.com/user/" + name; };
 
-    static auto getModLoaderStrings(ModLoaderType type) -> const QStringList
+    static auto getModLoaderStrings(const ModLoaderTypes types) -> const QStringList
     {
         QStringList l;
-        switch (type)
+        for (auto loader : {Forge, Fabric, Quilt})
         {
-            case Unspecified:
-                for (auto loader : {Forge, Fabric, Quilt})
-                {
-                    l << ModAPI::getModLoaderString(loader);
-                }
-                break;
-
-            case Quilt:
-                l << ModAPI::getModLoaderString(Fabric);
-            default:
-                l << ModAPI::getModLoaderString(type);
+            if (types & loader || types == Unspecified)
+            {
+                l << ModAPI::getModLoaderString(loader);
+            }
         }
+        if (types & Quilt && ~types & Fabric)  // Add Fabric if Quilt is in use, if Fabric isn't already there
```

Comment:
```suggestion
        if ((types & Quilt) && (~types & Fabric))  // Add Fabric if Quilt is in use, if Fabric isn't already there
```

It's hard to remember the precedence of those operators, so this makes it a bit more natural to read imo :slightly_smiling_face: